style.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background-color: #fff;
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: 0.5px;
+    height: 100%;
+  }
+  
+  header {
+    display: flex;
+    flex-direction: row;
+  }
+  
+  #brand {
+    margin-left: 20px;
+  }
+  .youtube {
+    height: auto;
+    width: 30px;
+  }
+  
+  .form-inline {
+    display: flex;
+    align-items: center;
+    margin-left: 470px;
+    /* margin-right: -200px; */
+  }
+  
+  .input-group {
+    width: 100%;
+    height: 70px;
+  }
+  
+  .youtube {
+    text-decoration: none;
+  }
+  .search-form {
+    border-radius: 30px 0 0 30px;
+  }
+  
+  .input-group-btn {
+    max-width: 90px;
+  }
+  
+  .search-btn {
+    color: rgb(26, 26, 36);
+    border-radius: 0 30px 30px 0;
+    background-color: #fff;
+    cursor: pointer;
+  }
+  
+  #video-player {
+    display: flex;
+    align-items: center;
+  }
+  
+  .main-video {
+    padding-left: 250px;
+    padding-top: -5px;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
The reason you described for a change is not professional enough to describe why this change is necessary so please consider always clarifying the reasons for a change means describing clearly to the user why the change is necessary or being asked.

Reply:
Than you for the feedback @ABDULALI3468 , I can see that my comments are definitely missing the reason and they can be misleading or hard to understand to a person I might send them to.